### PR TITLE
Update auto-merge.md

### PR DESCRIPTION
**Pull request by v-caxian**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@v-caxian : Thanks for your contribution! The author, @TinaLiuGirl, has been notified to review your proposed change.

File: CSIDevDocsPublic/v-caxian/auto-merge.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 title: "VS Code Authoring Extension"
 #description:
 #keywords:
-author: v-caxian
+author: TinaLiuGirl
```
Comment (PRMerger3) [inline CSIDevDocsPublic/v-caxian/auto-merge.md]:
```suggestion
author: V-CAXIAN
```

**Comment by PRMerger6** [comment]
@PRMerger3 : Thanks for your contribution! The author, @V-CAXIAN, has been notified to review your proposed change.

File: CSIDevDocsPublic/v-caxian/auto-merge.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 title: "VS Code Authoring Extension"
 #description:
 #keywords:
-author: v-caxian
+author: V-CAXIAN
 ms.author: v-caxian
 manager: arthurya
 ms.date:  10/29/2016
```
Comment (PRMerger16) [inline CSIDevDocsPublic/v-caxian/auto-merge.md]:
```suggestion
ms.date:  02/13/2019
```

**Comment by PRMerger14** [comment]
@PRMerger16 : Thanks for your contribution! The author, @V-CAXIAN, has been notified to review your proposed change.

**Pull request closed without merging**